### ISSUE TITLE
Freecam leaves the camera in a really messed up state if you dont select pause before entering it. (Crashes on OG)

### ISSUE THREAD

**Issue by Zedb0T** [open-goal/jak-project#3059]

**Comment by Zedb0T** [comment]
When I attempted to replicate this on OG, it was a consistent crash instead 🙃. 

Due to the fact that is observed when outside of debug mode I was not able to attach the debugger to get a log :(

**Comment by Zedb0T** [comment]
tbh, with the reboot in debug mode option available this could probably be considered super low prio, and maybe even just blocking free cam outside of full debug mode would be sufficient

**Comment by xTVaser** [comment]
There is no crash and it seems to work as it does on ps2?

https://github.com/open-goal/jak-project/assets/13153231/f7dd7448-ad1c-489f-bea1-fccc86ff96b8

**Comment by Zedb0T** [comment]
This is not what this issue is referencing. 


https://github.com/open-goal/jak-project/assets/89345505/a25f703c-9ea9-4fe1-96c5-75189d80e067

**Comment by Zedb0T** [comment]
[Uploading opengoal-support-package.zip…]()

**Comment by ManDude** [comment]
Yeah, definitely still a thing. I have no clue what could be causing this though.
